src/types/utils/index.ts

```diff
@@ -24,7 +24,7 @@ export type UnionOfProcedureFuncs<Args, ReturnValue, Storage> = Args extends unk
   : never;
 
 /**
- * Less strict version of Parameters<T>
+ * Less strict version of Parameters<T\>
```

Comment:
```suggestion
 * Less strict version of `Parameters<T>`
```

I thought we wanted to use `

Reply:
Ohh when I checked and generated docs, it altogether skipped <T> and only had `Less strict version of Parameters` so I thought all we wanted was to display it correctly